src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf

```diff
@@ -382,6 +382,10 @@
                 <source>Each element of this collection should satisfy its own set of constraints.</source>
                 <target>Mỗi phần tử trong bộ sưu tập này nên thỏa mãn những ràng buộc của nó.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid International Securities Identification Number (ISIN).</source>
+                <target>Giá trị này không phải là International Securities Identification Number (ISIN).</target>
```

Comment:
@van2412 Can you please have a look? I am not sure how we should translate "International Securities Identification Number". Thank you :-)

Reply:
I'm a Vietnamese person. I think it should be "Giá trị này không phải là Mã số chứng khoán quốc tế (ISIN).". Anyway, let's wait for your friend's response.

Reply:
@seriquynh This looks good to me. But I suggest adding the translation of "valid" as well. Like `Giá trị này không phải là Mã số chứng khoán quốc tế (ISIN) hợp lệ`

Reply:
I agree with your translation message.

Reply:
Thank you @seriquynh and @van2412